p2p/pex/pex_reactor.go

```diff
@@ -395,13 +395,35 @@ func (r *Reactor) ReceiveAddrs(addrs []*p2p.NetAddress, src Peer) error {
 
 // SendAddrs sends addrs to the peer.
 func (r *Reactor) SendAddrs(p Peer, netAddrs []*p2p.NetAddress) {
+	msg := filterAddrs(netAddrs)
```

Comment:
will we continue to send them all of our addresses still over time just not all in a single message?